uploader.json

```diff
@@ -4373,15 +4368,6 @@
                 }
             ]
         },
-        "lspci": {
-            "host": [
-                {
-                    "archive_file_name": "/insights_commands/lspci",
-                    "command": "/sbin/lspci",
-                    "pattern": []
-                }
-            ]
-        },
         "lssap": {
```

Comment:
@vishwanathjadhav please remove this change from your PR, this section of the json should not be changed.

Reply:
Done please check